proof/refine/ARM/KHeap_R.thy

```diff
@@ -3592,6 +3592,10 @@ lemma cur_tcb_cross:
   apply (erule (3) tcb_at_cross)
   done
 
+method add_cur_tcb' =
+  rule_tac Q="\<lambda>s'. cur_tcb' s'" in corres_cross_add_guard
+  , fastforce intro!: cur_tcb_cross
```

Comment:
I know that people have started with `;` at the beginning of lines
, which I also object to
, but`,` is really not an operator that should start a line
. The whole point of these operators being chosen as punctuation is that they should go with the preceding statement
, not the next statement
. If you imagine the same in English,
that would look entirely weird
.

Reply:
I totally agree with Gerwin here. Punctuations are...punctuations.